cartographer_rviz/cartographer_rviz/drawable_submap.cc

```diff
@@ -75,10 +80,16 @@ DrawableSubmap::DrawableSubmap(const ::cartographer::mapping::SubmapId& id,
                       ->getRootSceneNode()
                       ->createChildSceneNode()),
       submap_node_(scene_node_->createChildSceneNode()),
+      submap_id_text_node_(scene_node_->createChildSceneNode()),
       manual_object_(display_context->getSceneManager()->createManualObject(
           kManualObjectPrefix + GetSubmapIdentifier(id))),
       pose_axes_(display_context->getSceneManager(), scene_node_,
                  pose_axes_length, pose_axes_radius),
+      submap_id_text_(QString("(%1,%2)")
```

Comment:
Would it make sense to only set the string here and all other properties below? Right now the transform is split between the size here and the translation and alignment below.